tests/ui/extenv/extenv-not-env.rs

```diff
@@ -0,0 +1,6 @@
+// run-pass
+// Ensures that variables not defined through `--env` are stil available.
+
+fn main() {
+    assert!(!env!("PATH").is_empty());
```

Comment:
can you use `// rustc-env:` with some unique name from compiletest here instead of hoping that `PATH` happens to be defined?

Reply:
raced with @Urgau :D